packages/manager/src/features/CloudPulse/Utils/FilterConfig.ts

```diff
@@ -141,26 +141,16 @@ export const DBAAS_CONFIG: Readonly<CloudPulseServiceTypeFilterMap> = {
     },
     {
       configuration: {
+        dependency: ['resource_id'],
```

Comment:
```suggestion
        dependency: [RESOURCE_ID],
```

replace it with contant from contants file